dataeng/resources/warehouse-transforms-ci.sh

```diff
@@ -15,23 +15,18 @@ aws s3 cp s3://edx-dbt-docs/manifest.json ${WORKSPACE}/manifest
 # Setup to run dbt commands
 cd $WORKSPACE/warehouse-transforms
 
-# Pull the origin master code to latest branch which will be used to compare the diff
-git pull -f origin master:latest
+BRANCH_POINT=$(git merge-base origin/master ${ghprbActualCommit})
+git diff $BRANCH_POINT --name-only
```

Comment:
Is this line here just to write the diff to the console log for debugging purposes?

Reply:
That was my assumption, and it has proved handy to see what the actual diffs are in the logs.